cmd/sealos/cmd/run.go

```diff
@@ -99,6 +99,8 @@ func newRunCmd() *cobra.Command {
 	runCmd.Flags().BoolVar(&runSingle, "single", false, "run cluster in single mode")
 	runCmd.Flags().BoolVarP(&processor.ForceOverride, "force", "f", false,
 		"we also can input an --force flag to run app in this cluster by force")
+	runCmd.Flags().BoolVarP(&processor.Detach, "detach", "d", false,
+		"we run force mode in background,never prompt")
```

Comment:
-d seems to run some daemon..

Reply:
> -d seems to run some daemon..

Do you have any good suggestions? I don't think there are any words with better meanings.